ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -522,152 +476,132 @@ def check_backends(logger):
                 # Amount of objects to repair are rising
-                logger.success("No objects in objects repair queue")
-            elif cache["repair_percentage"] > repair_percentage:
+                logger.success('No objects in objects repair queue for backend_name {0}.'.format(backend_name))
+            elif cache[backend_name]['object_to_repair'] > objects_to_repair:
                 # Amount of objects to repair is descending
-                logger.failure("Amount of objects to repair is descending!")
-            elif cache["repair_percentage"] < repair_percentage:
+                logger.success('Amount of objects to repair is descending for backend_name {0}.'.format(backend_name))
+            elif cache[backend_name]['object_to_repair'] < objects_to_repair:
                 # Amount of objects to repair is rising
                 repair_rising = True
-                logger.failure("Amount of objects to repair are rising!")
-            elif cache["repair_percentage"] == repair_percentage:
+                logger.failure('Amount of objects to repair is rising for backend_name {0}.'.format(backend_name))
+            elif cache[backend_name]['object_to_repair'] == objects_to_repair:
                 # Amount of objects to repair is the same
-                logger.success("Amount of objects to repair are the same!")
+                logger.success('Amount of objects to repair is the same for backend_name {0}.'.format(backend_name))
 
             if logger.print_progress is False:
                 # Custom recap for operations
                 logger.custom(
-                    'Recap of disk-safety: {0}% of the objects have to be repaired. {1} backend(s) with disk lost and number of objects to repair are {2}.'
-                    .format(repair_percentage, len(backends_to_be_repaired),'rising' if repair_rising is True else 'descending or the same'),
-                    test_name,
-                    " ".join([str(repair_percentage), 'SUCCESS' if repair_rising == 0 else 'FAILURE', 'FAILURE' if repair_rising is True else 'SUCCESS']))
-            result["repair_percentage"] = repair_percentage
-            result["lost_backends"] = backends_to_be_repaired
-            CacheHelper.set(result)
-        return result
+                    msg='Recap of disk-safety: {0}% of the objects have to be repaired. Backend has lost {1} and number of objects to repair is {2}.'.format(repair_percentage, disk_lost, 'rising' if repair_rising is True else 'descending or the same'),
+                    test_name='{0}_{1}'.format(test_name, backend_name),
+                    value=' '.join([str(repair_percentage), 'SUCCESS' if disk_lost == 0 else 'FAILURE', 'FAILURE' if repair_rising is True else 'SUCCESS']))
+            if cache is None:
+                cache = {}
+            cache[backend_name] = {
+                    'object_to_repair': objects_to_repair,
+                    'total_objects': total_objects
```

Comment:
In this part of the code 'total_objects' is never used, so this can be removed
Or it could be used to print out the increment or decrement of the objects to repair if you base your calculation on percentages instead of absolute numbers